NAMESPACE

```diff
@@ -1564,6 +1564,7 @@ export(get_variance_residual)
 export(get_variance_slope)
 export(get_weights)
 export(has_intercept)
+export(has_one_value)
```

Comment:
Maybe `has_single_value()` is a better name? WDYT?

Reply:
Agreed (but can't modify now, I will change it later)